pkg/daemon/update.go

```diff
@@ -734,7 +754,17 @@ BOOT_ID=%s
 PENDING=%d`, pendingStateMessageID, pending.GetName(), dn.bootID, isPending)))
 
 	logger.Stdin = &pendingState
-	return logger.CombinedOutput()
+	loggerOutput, err := logger.CombinedOutput()
```

Comment:
I still think it'd be cleaner to call `loggerSupportsJournal` *once* or so on MCD startup rather than trying and failing each time.

Reply:
oh I can definitely do that if it's for `loggerSupportsJournal`

Reply:
should be fixed